sema4ai/bin/create_env/conda_vscode_darwin_amd64.yaml

```diff
@@ -19,3 +19,4 @@ dependencies:
           - sema4ai-http-helper==2.1.0
           - sema4ai-common==0.1.0
           - ruff==0.11.11
+          - fastmcp==2.10.5
```

Comment:
`fastmcp` has much more deps than needed which is too much for our simple usage. Please use `mcp` which at least has less deps.